docs/en/latest/getting-started/README.md

```diff
@@ -54,7 +54,13 @@ curl "http://127.0.0.1:9080" --head | grep Server
 If everything is ok, you will get the following response:
 
 ```text
-Server: APISIX/3.1.0
+Server: APISIX/Version
+```
+
+`Version` refers to the version of APISIX that you have installed. For example, APISIX 3.3.0.
+
+```text
+Server: APISIX/3.3.0
```

Comment:
The information is duplicated
![image](https://github.com/apache/apisix/assets/9354193/84da6477-9c80-43dd-a5d3-b94077fb04fb)

Reply:
fixed,I'll change it.
<img width="700" alt="image" src="https://github.com/apache/apisix/assets/76942195/946950d7-9640-435a-977c-781a5ba088ce">